test/unittests/execute_floating_point_test.cpp

```diff
@@ -407,6 +407,28 @@ TYPED_TEST(execute_floating_point_types, abs)
     }
 }
 
+TYPED_TEST(execute_floating_point_types, neg)
```

Comment:
Order the tests by opcode, if in the same fixture.

Reply:
Moved.